contrib/kml_export.lua

```diff
@@ -180,7 +244,7 @@ local function create_kml_file(storage, image_table, extra_data)
         return
     end
 
-    dt.print_error("Will try to export KML file now")
+    dt.print_log("Will try to export KML file now")
```

Comment:
I don't like the use of print_log here, because print_log is only available in the latest darktable version. There are still a lot of users with darktable 2.2 out there

Reply:
In the past we forked the Lua repo when doing a dt release, so for people using stable or old darktable versions there is a matching set of compatible scripts. Maybe we should bring that back?

Reply:
I think @houz has a good point.  Now that there is a windows port of DT we may have to introduce compatibility changes to get some of the scripts to run, such as using the new os detection feature.